plugins/GSdx/GSSetting.cpp

```diff
@@ -46,6 +46,7 @@ const char* dialog_message(int ID, bool* updateText) {
 #endif
 		case IDC_CRC_LEVEL:
 			return "Control the number of Auto-CRC hacks applied to games.\n\n"
+				"Automatic:\nAutomatically select 'Partial' or 'Full' level based on the selected renderer.\n\n"
```

Comment:
It should clearly state which option is automatically selected for each renderer. It's a bit vague now.